velox/common/memory/Memory.cpp

```diff
@@ -211,7 +211,11 @@ std::string MemoryManager::toString() const {
   out << "\t" << defaultRoot_->name() << "\n";
```

Comment:
Can you log the details of root pool as well? Thanks!

Reply:
Got it, done.